modules/meta.analysis/R/run.meta.analysis.R

```diff
@@ -6,129 +6,136 @@
+      !file.exists(file.path(pft$outdir, "prior.distns.Rdata"))) {
     logger.severe("Could not find output from get.trait for", pft$name)
     return(NA)
   }
-
+  
   # check to see if run.meta.analysis can be skipped
-  if (file.exists(file.path(pft$outdir, 'trait.mcmc.Rdata')) && 
-      file.exists(file.path(pft$outdir, 'post.distns.Rdata')) &&
-      settings$meta.analysis$update != TRUE ) {
+  if (file.exists(file.path(pft$outdir, "trait.mcmc.Rdata")) && 
+      file.exists(file.path(pft$outdir, "post.distns.Rdata")) && 
+      settings$meta.analysis$update != TRUE) {
     logger.info("Assuming get.trait copied results already")
     return(pft)
   }
-
+  
   # make sure there is a posteriorid
   if (is.null(pft$posteriorid)) {
     logger.severe("Make sure to pass in pft list from get.trait. Missing posteriorid for", pft$name)
     return(NA)
   }
-
+  
   # get list of existing files so they get ignored saving
-  old.files <- list.files(path=pft$outdir)
-
+  old.files <- list.files(path = pft$outdir)
+  
   logger.info("-------------------------------------------------------------------")
   logger.info(" Running meta.analysis for PFT:", pft$name)
   logger.info("-------------------------------------------------------------------")
   
   ## Load trait data for PFT
-  load(file.path(pft$outdir, 'trait.data.Rdata'))
-  load(file.path(pft$outdir, 'prior.distns.Rdata'))
-
-  if(length(trait.data) == 0) {
+  load(file.path(pft$outdir, "trait.data.Rdata"))
+  load(file.path(pft$outdir, "prior.distns.Rdata"))
+  
+  if (length(trait.data) == 0) {
     logger.info("no trait data for PFT", pft$name, "\n so no meta-analysis will be performed")
     return(NA)
   }
-
+  
   # create path where to store files
   pathname <- file.path(dbfiles, "posterior", pft$posteriorid)
   dir.create(pathname, showWarnings = FALSE, recursive = TRUE)
-
+  
   ## Convert data to format expected by pecan.ma
   jagged.data <- lapply(trait.data, jagify)
-
+  
   ## Check that data is consistent with prior
-  for(trait in names(jagged.data)){
-    data.median    <- median(jagged.data[[trait]]$Y)
-    prior          <- prior.distns[trait, ]
-    p.data         <- p.point.in.prior(point = data.median, prior = prior)
-    if(p.data <= 0.9995 & p.data >= 0.0005){
+  for (trait in names(jagged.data)) {
+    data.median <- median(jagged.data[[trait]]$Y)
+    prior       <- prior.distns[trait, ]
+    p.data      <- p.point.in.prior(point = data.median, prior = prior)
+    
+    if (p.data <= 0.9995 & p.data >= 5e-04) {
       if (p.data <= 0.975 & p.data >= 0.025) {
         logger.info("OK! ", trait, " data and prior are consistent:")
       } else {
         logger.warn("CHECK THIS: ", trait, " data and prior are inconsistent:")
       }
-    } else if (p.data > 0.9995 | p.data < 0.0005) {
-      logger.debug("NOT OK! ", trait," data and prior are probably not the same:")
+    } else if (p.data > 0.9995 | p.data < 5e-04) {
```

Comment:
@ashiklom these aren't any more magic than the log messages that they control - since there is no effect on the results I wouldn't worry too much; also, I don't think having a global P value for triggering debug messages is necessary (or even useful - since it will depend a lot on context).